### PR TITLE
Add Strong-name to EmbedIO assembly

### PR DESCRIPTION
**Pull request by geoperez**

### CODE REVIEW

File: src/EmbedIO/EmbedIO.csproj [hunk 1]
```diff
@@ -21,6 +21,13 @@
     <RepositoryType>git</RepositoryType>
     <PackageTags>webserver websocket webapi http</PackageTags>
     <LangVersion>7.3</LangVersion>
+    <Configurations>Debug;Release;Signed</Configurations>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Signed'">
```
Comment (rdeago) [inline src/EmbedIO/EmbedIO.csproj]: `<PropertyGroup Condition="'$(Configuration)' == 'Release' And Exists('NugetSigning.pfx')">`
...and you no longer need a Signed configuration.

File: src/EmbedIO/EmbedIO.csproj [hunk 2]
```diff
@@ -11,7 +11,7 @@
     <PackageId>EmbedIO</PackageId>
     <CodeAnalysisRuleSet>..\..\StyleCop.Analyzers.ruleset</CodeAnalysisRuleSet>
     <DebugType>Full</DebugType>
-    <Version>3.0.6</Version>
+    <Version>3.0.7</Version>
```
Comment (rdeago) [inline src/EmbedIO/EmbedIO.csproj]: This deserves its own commit IMHO.

**Comment by geoperez** [comment]
I will recreate the branch. Closing this.
